### PR TITLE
Switch to circleci.com

### PR DESCRIPTION
**Pull request by pvalsecc**

### CODE REVIEW

**Comment by pvalsecc** [comment]
I'm trying to figure out why circleci is not show in this PR...

File: circleci/dockercfg.enc [hunk 1]
```diff
@@ -0,0 +1 @@
+2�Y�w��G��c5�m�d7�j�����i�7>�;+�tc�@�����W|�+��&E�����Ο<`D9Fϑ#����ܔ	즦㒌��Odx�Jt�Eҧ��.'�U���G���bǩl��u�B�񧨛�ܨ|�
```
Comment (sbrunner) [inline circleci/dockercfg.enc]: It should be binary file ...

Comment (pvalsecc) [reply circleci/dockercfg.enc]: What do you want me to do?

Comment (pvalsecc) [reply circleci/dockercfg.enc]: `*.enc` is already set as binary: https://github.com/mapfish/mapfish-print/blob/master/.gitattributes#L15

Comment (sbrunner) [reply circleci/dockercfg.enc]: *.enc binary
In .gitattribute but it's already done, then I don't know, keep it like it.

Comment (pvalsecc) [reply circleci/dockercfg.enc]: But github doesn't care about gitattributes: https://stackoverflow.com/a/24382933/829301

Comment (sbrunner) [reply circleci/dockercfg.enc]: ok :-/

**Pull request merged**